packages/samples/headless-react/src/components/search-parameter-manager/search-parameter-manager.ts

```diff
@@ -1,30 +0,0 @@
-import {
```

Comment:
We should push people towards using the UrlManager to save them multiple headaches, so I implemented something similar to what Atomic is doing for the samples.

Reply:
Thanks, I agree these samples should be removed,